src/main/java/seedu/address/logic/parser/FindCommandParser.java

```diff
@@ -11,18 +12,22 @@
 /**
  * Parses input arguments and creates a new FindCommand object
  */
-public class FindCommandParser implements Parser<FindCommand> {
+public class FindCommandParser extends ArgumentMultimapParser<FindCommand> {
+
+    public FindCommandParser() {
+        super(COMMAND_SPECS);
+    }
 
     /**
      * Parses the given {@code String} of arguments in the context of the FindCommand
      * and returns a FindCommand object for execution.
      * @throws ParseException if the user input does not conform the expected format
      */
-    public FindCommand parse(String args) throws ParseException {
-        String trimmedArgs = args.trim();
+    public FindCommand parseArgumentMultimap(ArgumentMultimap argumentMultimap) throws ParseException {
+        String trimmedArgs = argumentMultimap.getPreamble();
```

Comment:
I'm quite confused about this line. Not sure how trimmedArgs is obtained from `getPreamble()`.

Reply:
Yup im dumb